### PR TITLE
Update auto-update config of onsen

### PR DESCRIPTION
**Pull request by asial-matagawa**
@PeterDaveHello 
This is a fixed version of the PR #9970.
Could you review this PR?

----
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/OnsenUI/OnsenUI
 * Official website (optional, not the repository): https://onsen.io/
 * NPM package url (optional): https://www.npmjs.com/package/onsenui
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3936
   - Count of watchers: 254
   - Count of forks: 537
 * NPM download stats (optional):
   - Downloads in the last day: 286
   - Downloads in the last week: 1142
   - Downloads in the last month: 4149

# Essential checklist
 * [x] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/onsen/package.json [hunk 1]
```diff
@@ -23,17 +23,13 @@
...
-        ]
-      }
-    ]
-  }
+  "npmName": "onsenui",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(js|css)/**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/onsen/package.json]: Should be `"+(js|css)/**/*onsen*"`, thanks.

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
I have fixed again. I am sorry for the bother...

**Comment by PeterDaveHello** [comment]
@asial-matagawa No worry!

**Comment by PeterDaveHello** [comment]
@asial-matagawa thanks for your contribution!

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
Thank you for the merging!

**Comment by PeterDaveHello** [comment]
@asial-matagawa Just found that I told you the wrong config, will fix from my side, it try to import external css files like font_awesome/css/font-awesome.min.css in the css which means we need to grab them all with +(css|js)/**/*

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
Actually I wrote that in https://github.com/cdnjs/cdnjs/pull/9970#discussion_r92581559, but I forgot to include your GitHub ID in the comment. I will be more careful. 

>will fix from my side
Thank you!

**Comment by PeterDaveHello** [comment]
@asial-matagawa oops, that's my fault that I didn't notice that

**Comment by PeterDaveHello** [comment]
@asial-matagawa if the css files are the same as upstream, maybe it's better to import them all from here, just separate them out

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
Thank you for suggestion, but the css files have to be loaded by `@import url(<relative path>);` due to the design of our library. Thank you.

**Comment by PeterDaveHello** [comment]
@asial-matagawa Yeah I'll host it, but actually you can just load them outside the library right?

**Comment by PeterDaveHello** [comment]
@asial-matagawa BTW can you guys provide the minified css file? Thanks.

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
Yes, it is technically possible but users can be confused because we have provided the following code as an official example until now (our library is basically used in offline environments such as [Cordova](http://cordova.apache.org/)).
```
<link rel="stylesheet" href="lib/onsen/css/onsenui.css"/>
<link rel="stylesheet" href="lib/onsen/css/onsen-css-components.css"/>
<script src="lib/onsen/js/onsenui.js"></script>
```
( https://onsen.io/v2/docs/guide/js/#loading-onsen-ui )

>BTW can you guys provide the minified css file? Thanks.
That's true, we would like to provide minified versions in the next release. Thank you!

**Comment by PeterDaveHello** [comment]
@asial-matagawa include the css files in another css in this case only makes the cdn content maintanence work harder and heavier, I'll suggest not to do that in future, most of the famous libs already been hosted here, you should just add few more lines about how to include them individually, thanks

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
OK, we will provide something like `onsenui-no-icons.css` to make our library friendly with CDNs.
Thank you for the important feedbacks!

**Comment by PeterDaveHello** [comment]
@asial-matagawa Thank you!

**Comment by PeterDaveHello** [comment]
@asial-matagawa https://www.keycdn.com/blog/front-end-optimization/
> Additionally, any webmasters mistakenly use the @import directive to include external stylesheets on a webpage. This is an outdated method, and it prevents browsers from performing parallel downloads. The link tag is your best option and will also improve the front end performance of your website. Furthermore, external stylesheets requested with the link tag do not block parallel downloads.

FYI :)

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
Thank you for the information!
We will use the correct method which you explained.

**Comment by PeterDaveHello** [comment]
@asial-matagawa Awesome!

**Pull request merged**